src/components/SpruceForm/FieldTemplates.tsx

```diff
@@ -91,9 +100,27 @@ const ArrayItem: React.FC<
       {children}
     </ExpandableCard>
   ) : (
-    <ArrayItemRow key={index}>
+    <ArrayItemRow key={index} border={border} index={index}>
+      {(hasMoveUp || hasMoveDown) && !readonly && (
```

Comment:
Why are we using `readonly` here instead of `isDisabled`?

Reply:
I figured there could be an instance where we'd want to show disabled buttons for an item that is disabled but not read-only. To me read-only definitely implies no interaction and I found the page was less cluttered hiding them, but I left them there when a form element is `disabled` for flexibility (these are two different settings in the library). They definitely could be hidden though.